### DISCUSSION TITLE
Hi, Dose ART support image segmentation models such as FCN? how to use it

### DISCUSSION THREAD

**Discussion by xiaoLeeeezi** [Trusted-AI/adversarial-robustness-toolbox#2223] [Q&A]

**Comment by beat-buesser** [comment]
Hi @xiaoLeeeezi I'll convert this issue into a Discussion to better answer the question.

**Reply by beat-buesser** [reply]
We are currently working on a new estimator for segmentation models in #2145, it is planned to be released with ART 1.16.
